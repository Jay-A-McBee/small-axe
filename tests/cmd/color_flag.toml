args = "tests/mock_dir -C"

[bin]
name = "tree"

[env]
inherit = false

[env.add]
LSCOLORS = "cxfxcxdxbxegedabagacad"

