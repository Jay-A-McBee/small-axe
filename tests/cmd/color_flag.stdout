"mock_dir"
â”œâ”€â”€â”€ [32mA[0m
â”‚    â”œâ”€â”€â”€ [32mA_NESTED[0m
â”‚    â”œâ”€â”€â”€ a.rs
â”‚    â”œâ”€â”€â”€ b.rs
â”‚    â””â”€â”€â”€ c.rs
â”œâ”€â”€â”€ A.rs
â”œâ”€â”€â”€ [32mB[0m
â”‚    â”œâ”€â”€â”€ a.rs
â”‚    â”œâ”€â”€â”€ b.rs
â”‚    â””â”€â”€â”€ c.rs
â”œâ”€â”€â”€ [32mC[0m
â”‚    â”œâ”€â”€â”€ a.rs
â”‚    â”œâ”€â”€â”€ b.rs
â”‚    â””â”€â”€â”€ c.rs
â””â”€â”€â”€ [35msym_linked -> "tests/mock_dir/A/"[0m


Total directories: 4 Total files: 10

